sunpy/map/mapsequence.py

```diff
@@ -271,6 +271,12 @@ def plot(self, axes=None, resample=None, annotate=True,
         plot_function : function
             A function to be called as each map is plotted.
             For more information see `sunpy.visualization.animator.MapSequenceAnimator`.
+        norm : string 
```

Comment:
It will be https://matplotlib.org/stable/api/cm_api.html#matplotlib.cm.ScalarMappable.set_norm

Reply:
A Colormap is an object, its type is not.

Reply:
Notice how they don't say object for the type of the first item in set_norm.

Reply:
There should be an example in our documentation: https://docs.sunpy.org/en/stable/generated/gallery/index.html

Reply:
Yes, getting closer. But cmap doesn't have to be a string. I am pretty sure that a norm doesn't have to be just an astropy class either.

Reply:
> hmm , but a same thing can have more than 1 dtpye ? .

Yes because the function we call to set both items, the colormap and normalization allows two types. As their documentation tells us.

> most of the times norm should be an astropy class because of the way it is used so dont you think this should work ?

It does not have to be an astropy class, therefore saying it can only be one is incorrect.

Reply:
It is times like this when going to the original function is very useful since otherwise knowing what types are allowed is very tricky.